### PR TITLE
[collections] append, not overwrite, EE volmounts

### LINKED ISSUE #764: Can't mount custom volume with ansible collections to execution environment
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
I am not able to add ansible collections to to execution environment when running ansible-navigator commands.

##### ANSIBLE-NAVIGATOR VERSION

```
ansible-navigator 1.1.0
```

Also I am on Fedora 34 and using podman.

##### CONFIGURATION

```yaml
---
ansible-navigator:
  execution-environment:
    image: registry.redhat.io/ansible-automation-platform-21/ee-supported-rhel8:1.0.0-30
    environment-variables:
      set:
        ANSIBLE_COLLECTIONS_PATHS: /home/jveverka/git/jwerak/ansible
    volume-mounts:
      - src: "/home/jveverka/git/jwerak/ansible"
        dest: "/home/jveverka/git/jwerak/ansible"
        label: "Z"
  logging:
    level: critical
    file: /tmp/log.txt
```

##### LOG FILE

Debug log output: https://raw.githubusercontent.com/jwerak/collection-test/master/log.txt

##### STEPS TO REPRODUCE

- Download the collection from https://github.com/jwerak/collection-test 
- Login to container registry registry.redhat.io
  - e.g. `podman login registry.redhat.io`
- Update ansible-navigator.yml to have base path relevant for your system
- List collections
  - `ansible-navigator collections --log-leve debug`

##### EXPECTED RESULTS

The command output displays the collections under my ansible collections base path.

##### ACTUAL RESULTS

No collections found.

##### ADDITIONAL INFORMATION

The collection for testing purposes is here: https://github.com/jwerak/collection-test

### PR DESCRIPTION
**Pull request by relrod**
Change:
- Ensure that when working in EEs, we don't forget our config-specified
  volmounts.

Test Plan:
- local testing

Tickets:
- Fixes #764

Signed-off-by: Rick Elrod <rick@elrod.me>

### CODE REVIEW

File: src/ansible_navigator/actions/collections.py [hunk 1]
```diff
@@ -253,13 +253,14 @@ def _build_plugin_content(self):
...
     def _run_runner(self) -> None:
+        # pylint: disable=too-many-branches
         """spin up runner"""
 
         if isinstance(self._args.set_environment_variable, dict):
             set_environment_variable = deepcopy(self._args.set_environment_variable)
         else:
             set_environment_variable = {}
-        set_environment_variable.update({"ANSIBLE_NOCOLOR": "True"})
+        set_environment_variable["ANSIBLE_NOCOLOR"] = "True"
```
Comment (cidrblock) [inline src/ansible_navigator/actions/collections.py:263]: thanks for doing these :)

**Review by cidrblock — APPROVED** [review]

**Comment by cidrblock** [comment]
I added the skip change log, I'm assuming this is related to the lint work coming because of the new way we handle these in the post-processor.... and not a bug today.

**Comment by relrod** [comment]
> I'm assuming this is related to the lint work coming because of the new way we handle these in the post-processor.... and not a bug today.

No, this fixes a bug today (see the referenced bug in the summary at the top). Currently, if you're using an EE, volmounts specified in the config won't get picked up in `ansible-navigator collections`. This fixes that.

**Review by cidrblock — APPROVED** [review]
awesome

**Pull request merged**